portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,51 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
```

Comment:
imports should be alphabetically sorted: https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing